src/components/Index.js

```diff
@@ -0,0 +1,50 @@
+import { createSuitMixin } from '../mixins/suit';
+import { createWidgetMixin } from '../mixins/widget';
+import indexWidget from 'instantsearch.js/es/widgets/index/index';
+
+// wrapped in a dummy function, since indexWidget doesn't render
+const connectIndex = () => indexWidget;
+
+export default {
+  name: 'AisIndex',
+  mixins: [
+    createSuitMixin({ name: 'Index' }),
+    createWidgetMixin({ connector: connectIndex }),
+  ],
+  provide() {
+    return {
+      // The widget is created & registered by widgetMixin, accessor is needed
+      // because provide is not reactive.
+      $_ais_indexWidget: () => this.widget,
+    };
+  },
+  props: {
+    indexName: {
+      type: String,
+      required: true,
+    },
+    indexId: {
+      type: String,
+      required: false,
+    },
+  },
+  render(createElement) {
+    return createElement(
+      'div',
```

Comment:
One notable difference with InstantSearch.js and React InstantSearch is that the `Index` widget is not render-less in Vue InstantSearch.

Is there a way to not render anything in Vue?

Reply:
yes, you can simply return the children. I can change that

Reply:
I've been looking, and it's not possible, because Vue only allows one child. What we can do instead is not rendering anything if there's no children.

Reply:
Worst case, having a plain `div` (without class names) is fine for now.